docs/user/advanced_features.rst

```diff
@@ -868,6 +868,316 @@ Constraint and PTO Features
+<https://github.com/WEC-Sim/WEC-Sim_Applications>`_ . The controller examples 
+are not comprehensive, but provide a reference for user to implement their 
+own controls.
+
+	+--------------------------------+-------------------------------------------+
+	|   **Controller Application**   |               **Description**             |                
+	+--------------------------------+-------------------------------------------+
+	|   Passive (P)  	         | RM3 Float with proportional control       |
+	+--------------------------------+-------------------------------------------+
+	|   Reactive (PI)                | Float with proportional-integral control  |
+	+--------------------------------+-------------------------------------------+
+	|   Latching		         | Float with latching control		     |
+	+--------------------------------+-------------------------------------------+
+	|   Declutching   		 | Float with declutching control            |
+	+--------------------------------+-------------------------------------------+
+	|   Model Predictive Control   	 | Float with model predictive control       |
+	+--------------------------------+-------------------------------------------+
+
+
+
+Examples: RM3 Float with Various Controllers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section explains the controller examples found within the WEC-Sim 
+Applications repository.
+
+First, it is important to understand the concept of complex conjugate control.
+Complex conjugate control, as applied to wave energy conversion, 
+can be used to understand optimal control. For a complex conjugate controller, 
+the impedance of the controller is designed to match the admittance of the 
+device which is equal to the complex conjugate of the impedance. Hence, it is 
+also known as impedance matching and is a common practice within electrical 
+engineering. Complex conjugate control is not a completely realizable control 
+method due to its acausality. which means it requires exact knowledge of 
+future wave conditions. Still, complex conjugate control presents a reference 
+for the implementation of optimal control. The WEC impedance can be modeled by 
+the following equation and can be used to formulate optimal control 
+implementation:
+
+.. math::
+
+    Z_i(\omega) = j\omega (I + I_{A}(\omega)) + R(\omega) + \frac{C_{hs}}{j\omega}
+
+By characterizing the impedance of the WEC, a greater understanding of the 
+dynamics can be reached. The figure below is a bode plot of the impedance of 
+the RM3 float body. The natural frequency is defined by the point at which the 
+phase of impedance is zero. By also plotting the frequency of the incoming 
+wave, it is simple to see the difference between the natural frequency of 
+the device and the wave frequency. Complex conjugate control (and many other
+control methods) seeks to adjust 
+the natural frequency of the device to match the wave frequency. 
```

Comment:
I would add a couple of comments about the challenges of adjusting the natural frequency of the device to match the wave frequency. In general, a resonant condition is not desire in any application, but this is not the case here, so it would be good to comment on this.